### ISSUE TITLE
list_of formating needs work

### ISSUE THREAD

**Issue by hadley** [r-lib/vctrs#206]
``` r
library(vctrs)
tibble::tibble(x = 1:2, y = list_of(data.frame(x = 1), data.frame(x = 1:10)))
#> # A tibble: 2 x 2
#>       x                             y
#>   <int>                <list<df[,1]>>
#> 1     1                             1
#> 2     2 1, 2, 3, 4, 5, 6, 7, 8, 9, 10
```

<sup>Created on 2019-02-27 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1.9000)</sup>

**Comment by hadley** [comment]
Just need to verify this wasn't fixed elsewhere.

**Comment by lionel-** [comment]
Fixed by @krlmlr in #317.
